Sources/LanguageServerProtocol/Requests/CallHierarchyIncomingCallsRequest.swift

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// The request is sent from the client to the server to resolve incoming calls for a given call hierarchy item.
+/// It is only issued if a server registers for the `textDocument/prepareCallHierarchy` request.
+public struct CallHierarchyIncomingCallsRequest: RequestType {
```

Comment:
Could we add a brief explanation of what "incoming" means here and similarly for the outgoing one?  Maybe it's just me, but I find "callers"  and "callees" much easier to understand.

Reply:
Changed